packages/medusa/src/models/cart.ts

```diff
@@ -213,8 +213,8 @@ export enum CartType {
 export class Cart extends SoftDeletableEntity {
   readonly object = "cart"
 
-  @Column({ nullable: true })
-  email: string
+  @Column({ type:'varchar', nullable: true })
```

Comment:
todo: Remove `type: 'varchar'`

Reply:
I remember getting errors without `type: 'varchar'`. I saw `type: 'varchar'` used in multiple other places so I assumed it was okay to do so.

Reply:
What errors did you experience?

Reply:
![image](https://user-images.githubusercontent.com/37419832/210240915-5755758d-3eb9-493b-8071-08cdc1209be9.png)